vector/src/debug/java/im/vector/app/features/debug/DebugMenuActivity.kt

```diff
@@ -265,6 +264,8 @@ class DebugMenuActivity : VectorBaseActivity<ActivityDebugMenuBinding>() {
                     }
                 }
             }
+            // Ensure developer will see that this cannot work anymore
+            error("toQrCodeData() is now internal")
```

Comment:
is the qrscanning debug option still helpful if we can't log the  `toQrCodeData` result?

Reply:
Probably not, hence not used a lot nowadays. I guess if we want to test it we can uncomment the code and remove the `internal` keyword. Not sure what was the best approach to avoid to do that for that specific case.